src/pages/zoro/zoro.ts

```diff
@@ -0,0 +1,42 @@
+import { malIdUpdated, selectMalId } from '../../data';
+import { BasePage } from '../base-page';
+import { Metadata } from '../base-page.types';
+import { ZoroSyncData } from './zoro.types';
+import metadata from './metadata.json';
+
+export class Zoro extends BasePage {
+  static getMetadata(): Metadata {
+    return metadata;
+  }
+
+  getSyncData(): ZoroSyncData {
+    return JSON.parse(document.getElementById('syncData')?.textContent ?? '{}');
```

Comment:
Oww, handy.
Actually how do you go on doing this? They seem to redirect you to home page when you open debugger.

Reply:
Add `zoro.to##+js(aopr, console.clear)` to your ublock origin filters.

It comes from https://github.com/Karmesinrot/Anifiltrs

(Tip from henrik9999)